index.html

```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+
+              </card_tags>
+            </li>
+          </card_works>
+
+          <card_works id="card_works">
+            <action id="action">
+              <img src="./assets/Mask Group.png" alt="">
+            </action>
+            <action_btn id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </action_btn>
+
+            <li id="card_data">
+              <card_title id="card_title"><p>Profesional Art Printing Data</p></card_title>
+              <card_secondary id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></card_secondary>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+
+              </card_tags>
+            </li>
+          </card_works>
+
+          <card_works id="card_works">
+            <action id="action">
+              <img src="./assets/Mask Group.png" alt="">
+            </action>
+            <action_btn id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </action_btn>
+
+            <li id="card_data">
+              <card_title id="card_title"><p>Profesional Art Printing Data</p></card_title>
+              <card_secondary id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></card_secondary>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+
+              </card_tags>
+            </li>
+          </card_works>
+
+          <card_works id="card_works">
+            <action id="action">
+              <img src="./assets/Mask Group.png" alt="">
+            </action>
+            <action_btn id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
```

Comment:
Kindly, I suggest you to use the "Button" tag, or "A" to very soon use it to display the details of your project.